src/main/scala/rps/Hello.scala

```diff
@@ -2,6 +2,7 @@ package rps
 
 object Hello extends Greeting with App {
```

Comment:
Ora che non è più un hello world, suggerirei di rinominare questo oggetto magari in `Main` o `Boot`.

(Lo so, non cambia niente in pratica, ma punterei in generale a tenere nomi consistenti, di solito noi ci aspettiamo uno di quei due nomi.)

Con l'occasione magari potresti anche eliminare l'uso di `Greeting` e mettere la stringa inline `println("...")`. Mi sembra una complicazione inutile del codice (mea culpa qui, sarebbe meglio avere un seed che non fa roba inutile).

Reply:
Ok